docs/releasing.md

```diff
@@ -1,8 +1,36 @@
-## Release guide for maintainers
+### Release guide for maintainers
+
+## Release Policy
+
+### Cadence
+
+We ship twice a month, on the Thursday before the 15th and the Thursday before 1st. These dates are added to the internal Lighthouse calendar.
```

Comment:
Why twice a month? My understanding was that we usually ship a big release once a month, and if needed ship a smaller release the second time around.

Reply:
I'm open to that. Seems okay for us to be more conservative and make for larger releases.

ok how about

> We ship once a month, on the Thursday before the 1st. While not necessary, followup minor/patch releases may be done if warranted.

cc @vinamratasingal @patrickhulce

Reply:
SGTM.